### ISSUE TITLE
pnext: 3levelschemachange_test and metadata assertions.

### ISSUE THREAD

**Issue by chrhol** [Starcounter/Starcounter#3292]
There is some commented out code in the 3levelschemachange_test that makes some assertions about metadata that does not currently work. These assertions were made based on the old metadata scheme and there seems to be some parts that are not correct. The metadata itself seems correct, but comes in a different order then the test expects.

I guess this code should be removed or rewritten. It's kind of hard to follow exactly what they test.

**Comment by k-rus** [comment]
Is it a duplicate of #3042?
